### ISSUE TITLE
Make @JsonDiscriminatorValue repeatable

### ISSUE THREAD

**Issue by mitasov-ra** [Tinkoff/kora#174]
At the moment I can put only one `@JsonDiscriminatorValue` on the class.

But OpenAPI allows to map multiple disriminators to one type via `mapping` setting like so:

```
ValidationErrorDto:
  title: ValidationErrorDto
  type: object
  oneOf:
    - $ref: '#/components/schemas/BlockValidationErrorDto'
    - $ref: '#/components/schemas/PropValidationErrorDto'
  discriminator:
    propertyName: type
    mapping:
      'Block.Error1': '#/components/schemas/BlockValidationErrorDto'
      'Block.Error2': '#/components/schemas/BlockValidationErrorDto'
      'Prop.Error1': '#/components/schemas/PropValidationErrorDto'
      'Prop.Error2': '#/components/schemas/PropValidationErrorDto'
```

With the above spec only last discriminators from mapping will be added on generated data classes.

**Comment by GoodforGod** [comment]
Related to #173

**Comment by GoodforGod** [comment]
Fixed in (#180)

**Comment by GoodforGod** [comment]
Released in `0.12.0`

**Comment by GoodforGod** [comment]
Look #173 for more info
